public/admin/config.yml

```diff
@@ -245,6 +245,13 @@ collections:
             max: 6
           - *meta_config
 
+      - name: "maturity-model"
+        label: "Maturity Model"
+        file: "src/content/pages/maturity-model.md"
```

Comment:
```suggestion
        file: "src/content/pages/security/maturity-model.md"
```

Reply:
Would be a good idea to look into amending `generatePageTemplate` in case we have subpages... 🤔

Reply:
I already left these comments in https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/548 so probably not relevant for this PR since the changes will be pulled from there, sorry!